nautobot/extras/templates/extras/customfield.html

```diff
@@ -0,0 +1,146 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load plugins %}
+{% load static %}
+{% load tz %}
+
+{% block header %}
+    <div class="row noprint">
+        <div class="col-sm-8 col-md-9">
+            <ol class="breadcrumb">
+                <li><a href="{% url 'extras:customfield_list' %}">Custom Fields</a></li>
+                {% if object.name %}
+                    <li><a href="{% url 'extras:customfield' pk=object.pk %}">{{ object.name }}</a></li>
+                {% endif %}
+            </ol>
+        </div>
+    </div>
+
+    <div class="pull-right noprint">
+        {% plugin_buttons object %}
+        {% if perms.extras.add_customfield %}
+            {% clone_button object %}
+        {% endif %}
+        {% if perms.extras.change_customfield %}
+            {% edit_button object %}
+        {% endif %}
+        {% if perms.extras.delete_customfield %}
+            {% delete_button object %}
+        {% endif %}
+    </div>
+
+    <h1>{% block title %}{{ object }}{% endblock %}</h1>
+    {% include 'inc/created_updated.html' %}
```

Comment:
Not applicable as this is not a ChangeLoggedModel; though this will be moot if you adopt the generic page template from #737.